tests/FormBuilderTest.php

```diff
@@ -44,6 +44,13 @@ public function testTextBox()
 		$this->assertEquals($expected, $result);
 	}
 
+	public function testNumberBox()
+	{
+		$expected = '<input type="number" name="age">';
+		$result = (string)$this->form->text('age');
```

Comment:
This one is failing because I believe you meant to call `number` not `text` :blush: